isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -2133,6 +2134,96 @@ namespace Isis {
   }
 
 
+/**
+ * Returns the latitude and longitude range for the Cube. More accurate than the minimum and 
+ * maximum latitude and longitude from the mapping group. 
+ * 
+ * @param minLatitude minimum latitude of the cube
+ * @param maxLatitude maximum latitude present in the cube
+ * @param minLongitude minimum longitude present in the cube
+ * @param maxLongitude maximum longitude present in the cube
```

Comment:
These should be flagged as output parameters. You can use `@param[out]` for this.